giphy-search/src/adapters/giphyAdapters-solution.js

```diff
@@ -0,0 +1,17 @@
+/* 
+An adapter is a helper function for sending a request to specific endpoint. Here, we have one adapter for fetching from the /trending endpoint and another for fetching from the /search endpoint. 
+*/
+import { API_KEY } from '../../config.js'
+import { handleFetch } from './handleFetch.js'
+
+const baseUrl = 'https://api.giphy.com/v1/gifs'
+
+export const getTrendingGifs = async () => {
+  const [data, error] = await handleFetch(baseUrl + `/trending?limit=3&rating=g&api_key=${API_KEY}`);
+  return [data.data, error];
+}
+
+export const getGifsBySearch = async (term) => {
+  const [data, error] = await handleFetch(baseUrl + `/search?limit=3&rating=g&api_key=${API_KEY}&q=${term}`);
+  return [data.data, error];
+}
```

Comment:
I think there are a few other ways that the fellows can solve this too.
A) They could simply return the tuple right away (instead of returning a tuple where `data.data` replaces `data`)
B) They could NOT use the `limit=3` query parameter and perform a slice of the fetched data here